cla_public/static-templates/errors/50x.html

```diff
@@ -121,51 +122,51 @@
         <div class="govuk-phase-banner">
           <p class="govuk-phase-banner__content">
             <span class="govuk-phase-banner__text" id="aria-feedback">
-              <a href="{{ url_for('base.reasons_for_contacting')}}">Report a problem</a> if you need help or would like to give feedback to improve this service
+              {% trans %}<a href="{{ url_for('base.reasons_for_contacting')}}">Report a problem</a> if you need help or would like to give feedback to improve this service{% endtrans %}
             </span>
           </p>
         </div>
         <main class="govuk-main-wrapper " id="main-content" role="main">
-          <h1 class="govuk-heading-xl">This service is temporarily unavailable</h1>
-          <p class="govuk-body">This service is not available right now, but we're working hard to get things up and running again.</p>
-          <p class="govuk-body">You can try again later or contact us. </p>
-          <p class="govuk-body">Telephone:<br />
+          <h1 class="govuk-heading-xl">{% trans %}This service is temporarily unavailable{% endtrans %}</h1>
+          <p class="govuk-body">{% trans %}This service is not available right now, but we're working hard to get things up and running again.{% endtrans %}</p>
+          <p class="govuk-body">{% trans %}You can try again later or contact us.{% endtrans %}</p>
+          <p class="govuk-body">{% trans %}Telephone:{% endtrans %}<br/>
             <strong class="govuk-!-font-weight-bold">0345 345 4345</strong>
           </p>
-          <p class="govuk-body">Minicom:<br />
+          <p class="govuk-body">{% trans %}Minicom:{% endtrans %}<br/>
             <strong class="govuk-!-font-weight-bold">0345 609 6677</strong>
           </p>
-          <p class="govuk-body">Textphone:<br />
+          <p class="govuk-body">{% trans %}Textphone:{% endtrans %}<br/>
             <strong class="govuk-!-font-weight-bold">0845 609 6677</strong>
           </p>
           <p class="govuk-body">
-            <a href="https://www.gov.uk/call-charges">Find out about call charges here</a>.
+            <a href="https://www.gov.uk/call-charges">{% trans %}Find out about call charges here{% endtrans %}</a>.
           </p>
-          <p class="govuk-body">Opening times:<br />
-            <strong class="govuk-!-font-weight-bold">Monday to Friday: 9am to 8pm<br />Saturday: 9am to 12.30pm</strong>
+          <p class="govuk-body">{% trans %}Opening times:{% endtrans %}<br />
+            <strong class="govuk-!-font-weight-bold">{% trans %}Monday to Friday: 9am to 8pm{% endtrans %}<br />{% trans %}Saturday: 9am to 12.30pm{% endtrans %}</strong>
           </p>
-          <div class="govuk-inset-text">
-            This service is only available if you are a resident of England or Wales.
+          <div class="govuk-inset-text">{% trans %}
+            This service is only available if you are a resident of England or Wales.{% endtrans %}
```

Comment:
Can we have this on one line?
`{% trans %}This service is only available if you are a resident of England or Wales.{% endtrans %}`

Reply:
For leading and trailing whitespace (ie this does not apply for the other comment, which addresses whitespace in between lines in the middle of a paragraph), I think that we can use `{%- trans -%}` to handle this. I might be wrong. Worth checking.